docs/src/main/sphinx/connector/hive-s3.rst

```diff
@@ -92,6 +92,22 @@ Property Name                                Description
                                              without staging file to be created in the local file system.
 
 ``hive.s3.streaming.part-size``              The part size for S3 streaming upload. Defaults to ``16MB``.
+
+``hive.s3.proxy.host``                       Proxy host to use if connecting through a proxy
+
+``hive.s3.proxy.port``                       Proxy port to use if connecting through a proxy
+
+``hive.s3.proxy.ssl.enabled``                Use HTTPS to communicate with Proxy, defaults to ``true``.
```

Comment:
It feels more like "should https:// request be handled by the proxy".
Maybe keep the nomenclature AWS uses (hive.s3.proxy.protocol)